dpgen2/superop/prep_run_fp.py

```diff
@@ -125,6 +125,8 @@ def _prep_run_fp(
     run_template_config = run_config.pop("template_config")
     prep_executor = init_executor(prep_config.pop("executor"))
     run_executor = init_executor(run_config.pop("executor"))
+    group_size = run_config.pop("group_size") if "group_size" in run_config else None
+    pool_size = run_config.pop("pool_size") if "pool_size" in run_config else None
```

Comment:
I would recommend put the two parameters to a section named `template_slice_config`, and pass the dictionary as extra kwargs to the class `Slices`. Then **any** extra parameters to the `Slices` is supported.